proposals/lip-0048.md

```diff
@@ -39,28 +39,29 @@ As introduced in [LIP 0013][lip-0013], all transactions must have a fee greater
 
 We use the definition of the following type:
 
-| Name | Type | Validation | Description |
-|------|------|------------|-------------|
+| Name          | Type   | Validation                                                                                                                                                                                                                                                                                    | Description                                          |
+| ------------- | ------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------- |
 | `Transaction` | object | Must follow the `transactionSchema` schema defined in [LIP 0068](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0068.md) with the only difference that `params` property is not serialized and contains the values of parameters of `paramsSchema` for the corresponding transaction. | An object representing a non-serialized transaction. |
+| `CCM`         | object | Must follow the `crossChainMessageSchema` schema defined in [LIP 0049](lip-0049#ccmschema).                                                                                                                                                                                                   | The type of cross-chain messages.                    |
```

Comment:
The link to the schema is broken. I suspect that `()` should be changed into `[]`. 
It looks like the link is resolved into the relative path `/lip-0049` from the location of the file, which is interesting.